frontend/test/metabase/scenarios/question/new.cy.spec.js

```diff
@@ -162,55 +162,55 @@ describe("scenarios > question > new", () => {
     });
   });
 
-  describe("ask a (simple) question", () => {
```

Comment:
Nice.

Reply:
That was a relic from the past. There are no simple and custom questions anymore.
Although, we still didn't deal with https://github.com/metabase/metabase/issues/19677

Reply:
Exactly, it's all in there and imo we should clean it up.